notebooks/254-llm-chatbot/254-llm-chatbot.ipynb

```diff
@@ -61,7 +61,7 @@
     "\"git+https://github.com/huggingface/optimum-intel.git\"\\\n",
```

Comment:
<p>&gt;&gt;please refer to the <a href="https://github.com/QwenLM/Qwen" target="_blank">GitHub</a> code repositoryel.</p><p> </p><p>not sure it is some formatting in editor issue or typo</p><p> </p><p>BTW, maybe chatglm3-6b also can be added?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1448/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
chatglm3 has more complicated input template, which is not compatible with our current template.